src/lime/_internal/backend/native/NativeApplication.hx

```diff
@@ -1,5 +1,6 @@
 package lime._internal.backend.native;
 
+import haxe.Int64;
```

Comment:
Why?

Reply:
Sometimes, when you type a class name and want to auto-import it, it inserts the wrong class, and auto imports it. Probably didn't pay enough attention, my bad. Nice catch!